src/it/kotlin/GivenTreepatExpressionBuildASTNodeTreeKtTest.kt

```diff
@@ -13,7 +13,7 @@ import antlr.treepat.TreepatParser
  */
 internal class GivenTreepatExpressionBuildASTNodeTreeKtTest {
 
-    private val resourcesLocation = "./src/it/resources/grammar.ast/"
+    private val resourcesLocation = "./src/it/resources/ast/"
```

Comment:
Also found this error, couldn't execute tests because of it. I'm not sure if you guys could run the tests or this line just got past us in our super professional last PRs.

Reply:
Nice!